### ISSUE TITLE
Test Auto Accessibility Detection in the Editor

### ISSUE THREAD

**Issue by alexandrudima** [Microsoft/vscode#27479]
Testing #14592 

- [x] mac with VoiceOver @isidorn 
- [x] windows with NVDA + [this plugin that increases the 30ms timeout]
 (https://github.com/Microsoft/vscode/issues/26730#issuecomment-303439633) @michelkaporin 

Complexity 3

We are now leveraging Electron APIs to detect when Electron/Chromium enables accessibility support. This is done in the main process and communicated to the renderer processes. Please check that:
* when running VS Code, the Ln, Col indicator is shown in the status bar and the `<textarea>` the editor uses for input/output is empty. You can find the `<textarea>` in the dom and use `$0.value` in the console.
* when starting NVDA / VoiceOver with VS Code running, all VS Code windows detect this and the status bar shows "Screen Reader Detected". The editor now begins to output screen reader content to the `<textarea>`. You can check this by finding the `<textarea>` in the dom and using `$0.value`. Also, the screen reader should now read the text in the editor. The Ln, Col indicator is removed because it is triggering accessibility events that confuses NVDA.
* please also check other scenarios, such as first starting the screen reader and then launching VS Code, or reloading a window, or opening a new window, etc. In all cases, the fact that a screen reader is attached should be detected. This is important, because otherwise the editor does not output screen reader content to the `<textarea>` (which gives us a nice performance boost)

**Comment by isidorn** [comment]
Works great
